core/errors.go

```diff
@@ -29,6 +30,8 @@ func isTemporary(info httperror.Info, err error) bool {
 		return true
 	case "CH001": // request timed out
 		return true
+	case "CH012": // conflict
+		return true
```

Comment:
How would this be used by clients? Generally I would expect that issuing the same request again would result in another conflict. If the client wants to retry their semantic operation, they'd have to do another read-modify-write, they can't just retry the same write. So I would think it should generally be permanent.

Or is the idea here that the entire read-modify-write cycle is happening inside of the HTTP handler? Then yeah the client's error would be temporary? But still I feel like in that situation, we should handle it in the server if possible. The default should be if ErrConflict is unhandled it should bubble up as a permanent error to the client (e.g. for /configure I think that would be the right thing if we weren't already explicitly checking for "already configured").